### ISSUE TITLE
Map nodes according to resource images

### ISSUE THREAD

**Issue by aaronmaynard** [aaronmaynard/Skyrim-Perk-Utility#19]
## 📦  Project: Ordinator Implementation

This issue is tied to the respective [project page](https://github.com/aaronmaynard/Skyrim-Perk-Utility/projects/1) and will be automatically sorted.  For more information on this project, [click here](https://github.com/aaronmaynard/Skyrim-Perk-Utility/tree/master/project/ordinator_implementation).   
❤️ Thanks for helping!

## 📋  Issue:

In order to use this on our graphical interface, we need to correctly map out the location of the nodes.  The layout is described as it is in our [resource images](https://github.com/aaronmaynard/Skyrim-Perk-Utility/tree/master/project/ordinator_implementation/resource%20images "Project resource files").  The nodes are positioned much like an x-y graph; x controlling horizontal movement, y controlling vertical.  Keep in mind however that the y position is inverted... so to go up by 10 points, you will need to enter -10.  Edit the [data_ordinator.js](https://github.com/aaronmaynard/Skyrim-Perk-Utility/blob/master/project/ordinator_implementation/data_ordinator.js) document found in the project directory. 

If you feel like you want to print out these pictures or trace them on graph paper to find the spacing, go for it! That is a great way to make sure everything is scaled correctly Here's a great [resource](http://incompetech.com/graphpaper/ "Online Graphing Paper").  If you prefer working on the computer you can use your preferred photo editing software while changing opacity of a graphing paper template.  If you do not own a copy of Photoshop, there is a great free online application called [Pixlr.com/editor](https://pixlr.com/editor/); requires Adobe Flash.

![Here](https://i.imgur.com/45ViEdq.jpg "Example")

Please keep the spacing on intervals of 5, such as `pos: [10, -10],` not `pos: [10, -13],`; this will make things easier / simpler.  The canvas will automatically readjust to the center-point.

```
 {
   name: 'Node X',			Change "X" to the new node number
   levels: 1,			        The number of levels the perk has
   desc: ['Description'],		The description of the perk
   req: [5],				The required skill level needed to activate this perk
   pos: [10, -10],			the position in intervals of 5
   deps: [X],				Change "X" to the node that a line will be drawn from.  For multiple lines, enter [X,X].
   id: ['00000000']			The code used to activate the perk in-game
 }
```

#### An recap of what you are doing:
- Set the name to "N# - [PerkName]"
- Map the node location on the canvas.
- That is it.

## 👥  Contributing:

-  Comment in this issue that you would like to do it.
-  Open the Skyrim-Perk-Utility GitHub page and click the ★ Star and then ⑂ Fork buttons.
-  Once you've made sure all your changes work correctly and committed all your changes,
-  Visit your fork on GitHub.com (https://github.com/YOUR-USER-NAME/Skyrim-Perk-Utility) and create a pull request for your changes.
-  Make sure your pull request describes exactly what you changed and references this issue (include the issue number in the title like this: #19)
-  Please <b>do not fix more than one issue at a time</b>. Your pull request should only fix what is described in this issue.

## 🆘  Asking for help

We appreciate your effort in taking the time to work on this issue and help out the open source community.
If you need any help, feel free to ask below. We are always happy to help 😄

**Comment by robatronPrime** [comment]
Hi can I contribute to this issue?

**Comment by aaronmaynard** [comment]
@robatronPrime Absolutely!  If you prefer using the computer, I suggest using the links above.  Please refer to the [Progress Table](https://github.com/aaronmaynard/Skyrim-Perk-Utility/tree/master/project/ordinator_implementation) for which ones need to be done, as well as reference if you get stuck.  Also a note to the example posted in this issue, I have found that once you graph the nodes on the perks, and they are too close, you may need to multiply them by 5. For example on the image above, the position for Kinship is `-5,-11`, but the end result is
```
{
   name: 'Kinship',
   levels: 1,
   desc: ['Buy items for 15% less when trading with the same race.'],
   req: [30],
   pos: [-25, -55],       // This is what was multiplied
   deps: [0],
   id: ['00000000']
},
```
Good luck!

**Comment by robatronPrime** [comment]
@aaronmaynard Hey, I've mapped two perk trees. How many would you like me to do?

**Comment by aaronmaynard** [comment]
@robatronPrime You can do as many as you like. 😄 Whatever you do not complete can be left for someone else to work on. It's a big project.  🤷‍♂️

**Comment by evalmay15** [comment]
Nodes mapped: Conjuration, Destruction, Alteration, Alchemy, Block, Archery. 👍 

We will need to reorder some nodes for dependencies, however they are all correctly mapped. 😄 And they look so pretty when testing! ❤️ 

We still need to map Illusion, Restoration, Enchanting, Smithing, Heavy Armor, One-Handed, Light Armor, Lockpicking and Pickpocket.  All of these have all nodes created so it should be easy. 🤔
